support/hydrationStyles.ts

```diff
@@ -0,0 +1,15 @@
+const fs = require("fs");
+const path = require("path");
+
+(async function () {
+  const stylePath = path.normalize(`${__dirname}/../src/assets/styles/_hydration.scss`);
+  const components = await fs.promises.readdir(path.normalize(`${__dirname}/../src/components/`));
+  const selectors = components.map((component) => `calcite-${component}:not([calcite-hydrated])`);
+  const fileContents = `// generated by support/hydrationStyles.ts
+${selectors.join(",\n")} {
+  visibility: hidden;
+  pointer-events: none;
+}`;
+  await fs.promises.writeFile(stylePath, fileContents);
+  process.exit(0);
+})();
```

Comment:
@paulcpederson ran this on a mac and I'm seeing some other lines added, such as...

```
// generated by support/hydrationStyles.ts
calcite-.DS_Store:not([calcite-hydrated])
calcite-calcite-accordion:not([calcite-hydrated]),
calcite-calcite-action-menu:not([calcite-hydrated]),
calcite-calcite-alert:not([calcite-hydrated]),
calcite-calcite-block-section:not([calcite-hydrated]),
calcite-calcite-button:not([calcite-hydrated]),
calcite-calcite-chip:not([calcite-hydrated]),
calcite-calcite-color:not([calcite-hydrated]),
calcite-calcite-color-picker:not([calcite-hydrated]),
calcite-calcite-combobox:not([calcite-hydrated]),
calcite-calcite-date-picker:not([calcite-hydrated]),
calcite-calcite-dropdown:not([calcite-hydrated]),
calcite-calcite-dropdown-group:not([calcite-hydrated]),
calcite-calcite-fab:not([calcite-hydrated]),
calcite-calcite-graph:not([calcite-hydrated]),
calcite-calcite-icon:not([calcite-hydrated]),
calcite-calcite-input:not([calcite-hydrated]),
calcite-calcite-input-date-picker:not([calcite-hydrated]),
calcite-calcite-input-message:not([calcite-hydrated]),
calcite-calcite-label:not([calcite-hydrated]),
calcite-calcite-list:not([calcite-hydrated]),
calcite-calcite-list-item:not([calcite-hydrated]),
calcite-calcite-modal:not([calcite-hydrated]),
calcite-calcite-pagination:not([calcite-hydrated]),
calcite-calcite-panel:not([calcite-hydrated]),
calcite-calcite-pick-list:not([calcite-hydrated]),
calcite-calcite-popover:not([calcite-hydrated]),
calcite-calcite-popover-manager:not([calcite-hydrated]),
calcite-calcite-radio-group:not([calcite-hydrated]),
calcite-calcite-shell:not([calcite-hydrated]),
calcite-calcite-shell-panel:not([calcite-hydrated]),
calcite-calcite-stepper:not([calcite-hydrated]),
calcite-calcite-tab:not([calcite-hydrated]),
calcite-calcite-tabs:not([calcite-hydrated]),
calcite-calcite-tile-select:not([calcite-hydrated]),
calcite-calcite-tile-select-group:not([calcite-hydrated]),
calcite-calcite-tooltip-manager:not([calcite-hydrated]),
calcite-calcite-tree:not([calcite-hydrated]),
calcite-calcite-tree-item:not([calcite-hydrated]),
calcite-calcite-value-list:not([calcite-hydrated]),
```

Reply:
Yeah, I just noticed a couple other ones, let me look at this a little closer

Reply:
ok, checking for only directories now, which should be more accurate.